wagtailio/project_styleguide/templates/patterns/components/streamfields/sponsor_block/sponsor_block.html

```diff
@@ -3,7 +3,7 @@
     <p class="sponsor-type">{{ value.sponsor_type }}</p>
     <div class="sponsor-logo">
         {% for logo in value.sponsor_logo %}
-            {% image logo width-500 %}
+            <a href="{{ value.sponsor_link }}">{% image logo width-500 %}</a>
```

Comment:
given `sponsor_link` is optional, this could use being wrapped in an `if` so we only include the `<a>` tag if we have the link